### ISSUE TITLE
Extracting chromosome and design type from EPIC V1 manifest in ctCheck.rmd

### ISSUE THREAD

**Issue by AnthonyKl98** [ejh243/BrainFANS#185]
### Contact Details

ak735@exeter.ac.uk

### What happened?

When running the QC for EPIC v1, I was getting the following error from the ctCheck.rmd. 

```
Error:
processing file: ./rmarkdownChild/ctCheck.rmd
Error in `[.data.frame`:
! undefined columns selected
Backtrace:
 1. ...[]
 2. base::`[.data.frame`(...)
Quitting from lines 3-87 [setupCtCheck] (./rmarkdownChild/ctCheck.rmd)
```

I think it relates to line 20:
`manifest<-manifest[match(fData(gfile)$Probe_ID, probeAnnot$probeID), c("CHR", "INFINIUM_DESIGN_TYPE")]`
Changing ProbeAnnot to manifest and "CHR", "INFINIUM_DESIGN_TYPE" to "chrm", "designType" (which match the column names of manifest) seemed to solve the problem i.e.
`manifest<-manifest[match(fData(gfile)$Probe_ID, manifest$probeID), c("chrm", "designType")]`

**Comment by sof202** [comment]
I think this requires a different kind of fix. I downloaded an epic v1 manifest file and the column names were different once again. Pretty sure that the column names have just changed over the years (unless I have downloaded the incorrect files: https://support.illumina.com/downloads/infinium-methylationepic-v1-0-product-files.html).

## Couple of obvious solutions

1) Ask the user for the proper names of the columns that we want
2) Use looser column matching (approximate match) to get the correct names of the columns
  - We know the approximate form of such columns, so that can help us to check if we have the correct 
  column extracted
3) Force everyone to use the same manifest file (awful decision).

There is probably a better solution beyond these. But we can't think of anything. I would like to give option 2 a go.

**Comment by sof202** [comment]
The solutions above suck. Anyways, this is implicitly fixed by #264.
